tests/test_shuffle.py

```diff
@@ -1,12 +1,59 @@
+import warnings
+
 import dask
+import pytest
+
+BASE_URL = "s3://dask-io/stability"
+storage_options = {"config_kwargs": {"region_name": "us-east-1"}}
+
+
+@pytest.fixture
+def shuffle_dataset(small_client):
+    # NOTE(sjperkins)
+    # I think the dataset and worker memory limits are related for this test
+    # as the dataset is large enough to provoke some spilling when it is
+    # persisted. Warn if the worker memory limits change egregiously.
+    # Alternatively, a separate cluster fixture may be appropriate
+    # for shuffle tests.
```

Comment:
Let's just fail if things fail.  I don't think that we should be this cautious.  (it also adds more code complexity than I think we want to add to every test.)

Reply:
I've removed this warnings